.github/workflows/run_type_checked_test_suite.yml

```diff
@@ -21,15 +21,16 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
-        python-version: "3.8"
+        python-version: "3.10"
     - name: Install dependencies
       run: |
         if [[ ${{ matrix.pytorch-version }} = "latest" ]]; then
-          pip install --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html;
+          pip install --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
         else
-          pip install torch==1.11+cpu -f https://download.pytorch.org/whl/torch_stable.html;
+          pip install torch==1.11+cpu -f https://download.pytorch.org/whl/torch_stable.html
+          pip install "numpy<2"
```

Comment:
why is numpy < 2 needed here?

Reply:
torch=1.11 is not compatible with numpy>2 but it does not explicitly require <2 either, so we end up with CI failing with `RuntimeError: Numpy is not available` on `tensor.numpy()` calls despite `numpy==2.2.2` being installed. See https://github.com/cornellius-gp/linear_operator/actions/runs/12961267021/job/36156321493

Reply:
I see. torch 1.11 is rather ancient but I guess this is only relevant for the CI testing, so this is fine by me.